Dockerfile

```diff
@@ -2,14 +2,15 @@ FROM centos:centos7
 MAINTAINER The ViaQ Community <community@TBA>
 
 EXPOSE 10514
-EXPOSE 24220
```

Comment:
This is probably just a coincidence but the port `24220` was prepared for fluentd `monitor_agent` feature. But anyway, I will add it again along with its `<source>` configuration as we discussed here https://github.com/ViaQ/docker-fluentd/pull/19#issuecomment-223589988